source/nodejs/adaptivecards-templating/package.json

```diff
@@ -17,7 +17,7 @@
 		"microsoft",
 		"bot"
 	],
-	"main": "dist/adaptivecards-templating.js",
+	"main": "lib/adaptivecards-templating.js",
```

Comment:
Why this change?

Reply:
Previously I added it to dist because the templating library was containing all the babel transpiled code in dist folder. Now as we decided to move the babel code to expressionlibrary itself, I have changed it to lib as it was before

Reply:
Ah, got it. I didn't realize you actually reversed a previous change.